encord/orm/label_row.py

```diff
@@ -200,6 +200,12 @@ class AnnotationTaskStatus(Enum):
     COMPLETED = "COMPLETED"
 
 
+class ShadowDataState(Enum):
```

Comment:
this can also have a small description. Doesn't have to be extensive.

Reply:
Are you still planning on adding something?

Reply:
Just pushed the docs for this thing, but otherwise no, I feel like I'm done